README.md

```diff
@@ -18,12 +18,23 @@ Destination path prefix where files will be stored
 
 Note: Path must start with a leading /
 
+#### `timestamp`
+Boolean. If true will create month folder in dstPath and append timestamp to dstPath
+
+Default: false
+
+Will use final directory in dstPath as project name
+
+Example path given date of April 3rd 2020 at 09:30: `destination/path/2020-04/projectName_2020-04-03_09-30/`
```

Comment:
This flag removes the final directory in dstPath and uses this as the projectName.  This could be confusing so I can look into changing this behaviour in another update